disconnected/mirroring/about-installing-oc-mirror-v2.adoc

```diff
@@ -8,49 +8,31 @@ toc::[]
 
 :FeatureName: oc-mirror plugin v2
 include::snippets/technology-preview.adoc[]
 
+// About oc-mirror plugin v2
+include::modules/oc-mirror-v2-about.adoc[leveloffset=+1]
+
+// oc-mirror compatibility and support
+include::modules/oc-mirror-v2-support.adoc[leveloffset=+2]
+
 [id="prerequisites_oc-mirror-v2_{context}"]
 == Prerequisites
 
 * You must have a container image registry that supports link:https://docs.docker.com/registry/spec/manifest-v2-2[Docker V2-2] in the location that hosts the {product-title} cluster, such as {quay}.
 +
 [NOTE]
 ====
-If you use {quay}, use version 3.6 or later with the oc-mirror plugin. See the documentation on link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html/deploying_the_red_hat_quay_operator_on_openshift_container_platform/index[Deploying the Red Hat Quay Operator on OpenShift Container Platform (Red Hat Quay documentation)]. If you need additional assistance selecting and installing a registry, contact your sales representative or Red Hat Support.
-====
+* If you use {quay}, use version 3.6 or later with the oc-mirror plugin. See link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html/deploying_the_red_hat_quay_operator_on_openshift_container_platform/index[Deploying the Red{nbsp}Hat Quay Operator on OpenShift Container Platform (Red{nbsp}Hat Quay documentation)]. If you need additional assistance selecting and installing a registry, contact your sales representative or Red{nbsp}Hat Support.
 
-[Optional]
-* If you do not have an existing solution for a container image registry, {product-title} subscribers receive a mirror registry for Red Hat OpenShift. This mirror registry is included with your subscription and serves as a small-scale container registry. You can use this registry to mirror the necessary container images of {product-title} for disconnected installations.
+* If you do not have an existing solution for a container image registry, {product-title} subscribers receive a mirror registry for Red{nbsp}Hat OpenShift. This mirror registry is included with your subscription and serves as a small-scale container registry. You can use this registry to mirror the necessary container images of {product-title} for disconnected installations.
+====
 
 * Every machine in the provisioned clusters must have access to the mirror registry. If the registry is unreachable, tasks like installation, updating, or routine operations such as workload relocation, might fail. Mirror registries must be operated in a highly available manner, ensuring their availability aligns with the production availability of your {product-title} clusters.
 
-.High level workflow
```

Comment:
Moved this section to the about section: https://github.com/openshift/openshift-docs/pull/84069/files#diff-5e72668c367f748ce622ac7f5914e485b7c7a4803986b5ddfd954fda7663a218